core/domain/email_manager.py

```diff
@@ -255,3 +297,57 @@ def send_moderator_action_email(
     _send_email(
         recipient_id, sender_id, intent, email_subject, full_email_content,
         bcc_admin=True)
+
+def send_exploration_membership_email(
+        sender_id, recipient_id, recipient_role, exploration_id,
```

Comment:
Consider changing sender_id --> inviter_id. The reason I say this is because, technically speaking, it's the Oppia system that sends the email, not the sender_id.